pkg/plugins/helm/v1/api.go

```diff
@@ -127,6 +137,13 @@ func (p *createAPISubcommand) InjectConfig(c config.Config) error {
 	return nil
 }
 
+func (p *createAPISubcommand) PreScaffold(machinery.Filesystem) error {
```

Comment:
@camilamacedo86 Thanks for looking into this. Just a follow-up, we may also have to look into hybrid plugin to see if we are still supporting `CRDVersion`. I think we are not, but it would be helpful to fix that in this PR itself, in case we are.

Reply:
All plugins should deprecate that since it is no longer useful from k8s 1.22 and we will probably remove that in the future versions.
Could we do that in a follow-up? @Venkat19967 that seems small one ... could you give to us a hand here?